### PR TITLE
Added a new Field in FormBuilder to support Radio Groups

### PR DESCRIPTION
**Pull request by wshamim**

### CODE REVIEW

File: src/components/TreeActionPanel/FormBuilder.js [hunk 1]
```diff
@@ -105,6 +105,12 @@ export default class FormBuilder {
         res.maxColumns = defField.maxColumns;
       }
     }
+    if (defField.type == 'Radio') {
```
Comment (michaeljharrison) [inline src/components/TreeActionPanel/FormBuilder.js]: Should we be using === here for good practice?

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json [hunk 2]
```diff
@@ -3,18 +3,25 @@
...
+      "tooltip": "Apply filter operator across filter conditions",
+      "type": "Radio",
+      "defaultValue": "none",
+      "radios": [
+        {
+          "label": "none",
+          "value": "none"
+        },
+        {
+          "label": "Use Or?",
```
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Use $or"

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json [hunk 3]
```diff
@@ -3,18 +3,25 @@
...
+          "value": "UseOr"
+        },
+        {
+          "label": "Use And?",
```
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Use $and"

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json [hunk 4]
```diff
@@ -3,18 +3,25 @@
...
-      "type": "Boolean",
-      "default": false
+      "name": "filter",
+      "label": "Filter Operator",
+      "tooltip": "Apply filter operator across filter conditions",
+      "type": "Radio",
+      "defaultValue": "none",
+      "radios": [
+        {
+          "label": "none",
```
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Default"

**Review by michaeljharrison — CHANGES_REQUESTED** [review]
Functionally looks good, few minor changes to make before we merge it, I've added comments to the relevant sections but here is an overall:

- Ability to add Tooltips for individual radio elements.
- Swap SimpleQuery Or and And fields to this new radio button component.
- Get styling approved / created by Kalid, if he's happy with it I'm happy with it.

**Comment by wshamim** [comment]
I have incorporated the changes which you have requested. Have a look now and let me know if anything else is required.

**Review by michaeljharrison — APPROVED** [review]
Looks good.

**Pull request merged**